### PR TITLE
Kmt 187 kmt myft email attribution

### PR DESCRIPTION
**Pull request by Thurst-ft**
Updates myft to assign segmentIds to topic and article links where KMT was used to 'follow' said topic.

### CODE REVIEW

File: functions/create-email/views/partials/concept-url.html [hunk 1]
```diff
@@ -1 +1 @@
-https://www.ft.com/__opt-in?optedvia=myft-email&amp;referrer={{encode url}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '&segmentId='}}{{@root.segmentId}}{{/if}}&amp;hash=myft:notification:{{@root.type}}-email:tag
\ No newline at end of file
+https://www.ft.com/__opt-in?optedvia=myft-email&amp;referrer={{encode url}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '&segmentId='}}{{#if katAttribution}}{{@root.katSegmentId}}{{else}}{{@root.segmentId}}{{/if}}{{/if}}&amp;hash=myft:notification:{{@root.type}}-email:tag
```
Comment (keirog) [inline functions/create-email/views/partials/concept-url.html]: The newline needs to be removed, sorry

File: functions/create-email/views/partials/content-url-plain.html [hunk 2]
```diff
@@ -1 +1 @@
-https://www.ft.com/__opt-in?optedvia=myft-email&referrer={{encode 'https://www.ft.com/content/'}}{{encode id}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '?segmentId='}}{{@root.segmentId}}{{/if}}&hash=myft:notification:{{{@root.type}}}-email:content
\ No newline at end of file
+https://www.ft.com/__opt-in?optedvia=myft-email&amp;referrer={{encode 'https://www.ft.com/content/'}}{{encode id}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '?segmentId='}}{{#if katAttribution}}{{@root.katSegmentId}}{{else}}{{@root.segmentId}}{{/if}}{{/if}}&amp;hash=myft:notification:{{{@root.type}}}-email:content
```
Comment (keirog) [inline functions/create-email/views/partials/content-url-plain.html]: Same newline issue

File: functions/create-email/views/partials/content-url.html [hunk 3]
```diff
@@ -1 +1 @@
-https://www.ft.com/__opt-in?optedvia=myft-email&amp;referrer={{encode 'https://www.ft.com/content/'}}{{encode id}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '&segmentId='}}{{@root.segmentId}}{{/if}}&amp;hash=myft:notification:{{{@root.type}}}-email:content
\ No newline at end of file
+https://www.ft.com/__opt-in?optedvia=myft-email&amp;referrer={{encode 'https://www.ft.com/content/'}}{{encode id}}{{encode '?desktop=true'}}{{#if @root.segmentId}}{{encode '&amp;segmentId='}}{{#if katAttribution}}{{@root.katSegmentId}}{{else}}{{@root.segmentId}}{{/if}}{{/if}}&amp;hash=myft:notification:{{{@root.type}}}-email:content
```
Comment (keirog) [inline functions/create-email/views/partials/content-url.html]: Same newline issue

**Comment by Thurst-ft** [comment]
@keirog no more newlines. ST3 rather annoyingly still inserts them even if you  have "ensure_newline_at_eof_on_save": false in the settings.

File: functions/create-email/views/partials/content-url-plain.html (hunk 2 above)
Comment (keirog) [inline functions/create-email/views/partials/content-url-plain.html]: Could you swap the `&amp;`s back to `&` in this file please? It’s for plain-text emails, so no HTML entities allowed (apologies for the misleading file extension).

**Comment by keirog** [comment]
> No change to content-url-plain.html?

Sorry, I meant `concept-url-plain.html`. Could we amend that file too please?

**Pull request merged**